modules/cnvpytor/histogram/main.nf

```diff
@@ -0,0 +1,29 @@
+process CNVPYTOR_HISTOGRAM {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::cnvpytor=1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cnvpytor:A1.0--py39h6a678da_2':
+        'quay.io/biocontainers/cnvpytor:1.0--py39h6a678da_2' }"
+
+    input:
+    tuple val(meta), path(pytor)
+
+    output:
+    tuple val(meta), path("${pytor.baseName}.pytor")	, emit: pytor
+    path "versions.yml"                                 , emit: versions
+
+    script:
+    def args = task.ext.args ?: '1000'
```

Comment:
@projectoriented : In the histogram and partition modules, I have to use the .baseName feature to be able to address the input file as the output. So I'm not calling args or prefix variables here. If you remember what we discussed that these two modules write output into the input file. But I can use 'mv' to copy generated output to a new file s.th. like $prefix.pytor.
Do you think it is necessary?

Reply:
Ohhh right yes, I think it should be okay without. Hmm I also found this `-o`  & wonder if this can be used here https://github.com/abyzovlab/CNVpytor/blob/master/GettingStarted.md#visualization 🤔